src/url.utils.ts

```diff
@@ -1,7 +1,8 @@
 import { config } from './config';
 
 export const validateUrl = (url: string) => {
-  if (config().isTestEnvironment) {
+  const x = config();
```

Comment:
We don't use such names. You can import `config` under different name and use `config` as variable name.

Or just make `config` static, I don't know why it's a function.

@JonaszJestem  maybe can shed some light on this

Reply:
Shame on me, it left after debug.

Reply:
I made config an object.